lua/xit/init.lua

```diff
@@ -296,7 +296,7 @@ local configured = false
 local M = {}
 
 M.setup = function(opts)
-  options = vim.tbl_deep_extend('force', options, opts)
+  options = vim.tbl_deep_extend('force', options, opts or {})
```

Comment:
Looks like a different way to fix it as in https://github.com/synaptiko/xit.nvim/pull/4/files